internal/pkg/config/v1/config.go

```diff
@@ -134,7 +135,8 @@ func New() *Config {
 }
 
 func (c *Config) DecryptContextStates() error {
-	for _, context := range c.Contexts {
+	if c.CurrentContext != "" {
+		context := c.Contexts[c.CurrentContext]
```

Comment:
This should be equivalent!
```suggestion
		context := c.Context()
```